development/button-app/pages/button/index.mdx

```diff
@@ -27,43 +38,76 @@ It uses following tailwind classes for providing styles: ** ``` w-1/2 bg-red-400
 ```
 
 ## Sample Code Demo
-
+This is the sample styles included when you return **ReuseButton** as shown in the demo code. It has the features of setting the **busy state** and the **busy text** which will be visible when in **busy state**.
 <Sample/>
 
 ```js path=../../components/button/Sample name=Sample
 ```
 
 ## Outline Button Demo
-
+This is a simple ReuseButton with the styling of **border-2 border-blue-600 bg-transparent font-bold text-blue-600 py-1 px-2** to give it a **transparent button** look with outlines. It can be suitable for situations where a more subdued visual appearance is desired while still maintaining the interactive nature of a button.
 <OutlineButton/>
 
 ```js path=../../components/button/OutlineButton name=OutlineButton
 ```
 
 ## Success Sample Button Demo
-
+The Success Sample Button is a customizable button that provides feedback to the user by showing a **loading** state for a specified duration and then disabling the button and displaying a **success** state. The **success** state and the **busy** state along with the **duration** can be customised as per the requirement.
 <SuccessSample/>
 
 ```js path=../../components/button/SuccessSample name=SuccessSample
 ```
 
 ## Text Button Demo
-
+This is a simple button with no visible outlines to make it look minimalistic. It can be used to embed **hyperlinks** within your text to make it standout.
 <TextButton/>
 
 ```js path=../../components/button/TextButton name=TextButton
 ```
 
 ## CSSStyledButton
-
+This code snippet of the button will give you a customizable button with CSS Styling. We have declared a styles object having CSS properties which can be passed as a prop to our HeadlessButton component to give it the style we need. You can customize the styles object according to your requirements.
```

Comment:
`passed as a prop to our HeadlessButton component`
This part is not necessary as this is the default way that styling is done in any React/Html component.